### PR TITLE
media-tv/kodi/kodi: version bump to 17.0_rc4, remove "dvd? dev-libs/libcdio[-minimal]" dependency

### PR DESCRIPTION
**Pull request by candrews**
https://bugs.gentoo.org/show_bug.cgi?id=605578

### CODE REVIEW

File: media-tv/kodi/kodi-17.0_rc4.ebuild [hunk 1]
```diff
@@ -56,7 +56,6 @@ COMMON_DEPEND="${PYTHON_DEPS}
 	>=dev-libs/yajl-2
 	dev-python/pillow[${PYTHON_USEDEP}]
 	dev-libs/libcdio
-	dvd? ( dev-libs/libcdio[-minimal] )
```
Comment (SoapGentoo) [inline media-tv/kodi/kodi-17.0_rc4.ebuild]: no revbump necessary -> `-N` will trigger a rebuild if you had `USE=dvd` enabled

Comment (anthonyryan1) [inline media-tv/kodi/kodi-17.0_rc4.ebuild]: This isn't the cause of a revbump, it's just queued work being done on the current version bump.

Comment (SoapGentoo) [inline media-tv/kodi/kodi-17.0_rc4.ebuild]: What does "queued work" mean? The only file change I see is the `rm -rf` below. The VDB change for removing `dev-libs/libcdio` will be triggered by `--new-use`/`-N`

Comment (candrews) [inline media-tv/kodi/kodi-17.0_rc4.ebuild]: The ebuild is renamed from rc3 to rc4. There is a version bump (not a rev bump).

**Pull request closed without merging**